spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Users requests', type: :request do
+  describe 'Index method' do
+    it 'returns the users list on the root' do
+      get root_path
+      expect(response).to have_http_status(200)
+    end
+
+    it 'returns the users list on normal path' do
+      get users_path
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template in both' do
+      get users_path
+      expect(response).to render_template(:index)
+
+      get root_path
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe 'Show method' do
+    before(:example) { get user_path(id: 1) }
+
+    it 'returns the data from the correct user' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template(:show)
+    end
+  end
```

Comment:
- [ ] I guess you forgot to write the test that will check the **response body if it includes a correct placeholder text.**

Both actions `index` and `show` should have a response body test

See example:

```ruby
it 'renders the correct placeholder text' do 
     expect(response.body).to include("Welcome to user index page")
end
```

Reply:
I believe you refer to those tests? 
![image](https://user-images.githubusercontent.com/90217257/171942448-cc822d8f-7c2c-40f9-9eff-e59c08e563ae.png)
 They are in the path indicated on top of the image, in the views folder. I am sorry I did not make it clear.
I also do not know how to highlight specific lines like you do in the issue. I will look it up.

Reply:
Its this file spec/views/posts_view_spec.rb

Reply:
Good to point them out. Currently, you are doing request tests, and it is good to put them in the `/requests/<test-file-name>`

Reply:
So, just to be clear, In order to approve me you are requiring that I change the name of the folder containing the tests?

Reply:
Mr. @mmsesay if you could please kindly clarify whether I am fulfilling the requirements of the projects or not, since I am already quite late on the deliveries this week.